api/apiFunctions.test.tsx

```diff
@@ -21,297 +19,375 @@ jest.mock('./apiFunctions', () => {
     getAllLeagues: jest.fn(),
     getUserDocumentId: jest.fn(),
     addUserToLeague: jest.fn(),
+    getCurrentLeague: jest.fn(),
+    loginAccount: jest.fn(),
```

Comment:
I don't see this being used anywhere in this test file, and if we're removing the `loginAccount` function from `apiFunctions`, what is this mocking?